src/DelayedAdmin.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
I would move `DelayedAdmin` and `PauseAdmin` to a new `src/admin` folder to organize it a bit more